services/galley/src/Galley/Data/Types.hs

```diff
@@ -99,3 +101,9 @@ data LockAcquired
   = Acquired
   | NotAcquired
   deriving (Show, Eq)
+
+data MemberAddStatus = MemberAddStatus
```

Comment:
_note to self: please add this comment to the docs and make sure it's considered in the logic._

we can't just fail to the requesting client with "the following domains could not be reached", because a remote site may go down after adding some of its members to a conversation, but before adding the rest.  so clients need to be informed about failures of adding individual members.

`MemberAddStatus` is confusing me.  i don't think we're using `added` anywhere in either of these two PRs.  maybe this is about the above, and `added` should be renamed to `notNotified`?

Reply:
There are two parts to your comment.

I believe we addressed the first one: the federation API does not work at this level of granularity, so there's no incremental update to the conversation-owning backend on the progress of member adding on a remote backend's end.

As to the second part regarding `MemberAddStatus`, I believe you've already addressed that with your changes.

I'm marking this comment as resolved, but revert that if there's more to it.